nvflare/private/fed/client/client_executor.py

```diff
@@ -214,6 +185,18 @@ def start_app(
         )
         thread.start()
 
+    def _get_job_launcher(self, job_id, client, job_meta: dict) -> dict:
+        launch_image = extract_job_image(job_meta, client.client_name)
+        launch_data = {JobConstants.JOB_IMAGE: launch_image, JobConstants.JOB_ID: job_id}
+
+        job_launcher = None
+        for _, component in client.components.items():
```

Comment:
We shouldn't need to call "can_launch" if we use event.

Reply:
can_launch() has been removed.